stabilizer/src/main/java/com/hazelcast/stabilizer/worker/tasks/SingleOperationAbstractWorkerTask.java

```diff
@@ -0,0 +1,33 @@
+package com.hazelcast.stabilizer.worker.tasks;
+
+/**
+ * Single operation version of {@link AbstractWorkerTask}.
+ * <p/>
+ * There is no {@link com.hazelcast.stabilizer.worker.selector.OperationSelector} used in this worker.
+ */
+public abstract class SingleOperationAbstractWorkerTask extends AbstractWorkerTask {
+
+    public SingleOperationAbstractWorkerTask() {
+        super();
+    }
+
+    @Override
+    public void run() {
+        beforeRun();
+
+        while (!testContext.isStopped()) {
+            doRun();
+
+            increaseIteration();
+        }
+        operationCount.addAndGet(iteration % performanceUpdateFrequency);
+
+        afterRun();
+    }
+
+    @Override
+    protected final void doRun(Enum operation) {
+    }
+
+    protected abstract void doRun();
```

Comment:
Can we rename this to runIteration? The run does the loop and the doRun does not.. I find this confusing. 

With 'runIteration' you know that only a single iteration is done..